### PR TITLE
Feature/chat

### PR DESCRIPTION
**Pull request by ArtemAng**
I've made a view for rooms and I've also bound Socket's joinRoom to rooms in database. 
If you want to test it, you can use the same route ( /chat/ ), but you alse need to change your token in the file views/room.html.

### CODE REVIEW

**Review by danielDeVita — APPROVED** [review]

File: src/views/chat/all-rooms.html [hunk 1]
```diff
@@ -1,14 +1,41 @@
...
+            roomDiv.innerHTML = `
+              <li> <a href="/room/${room._id}">${room._id}</a></li>
+            `;
+            roomsListElement.appendChild(roomDiv);
+          });
+        } else {
+          console.error('Ошибка получения комнат:', response.status);
+        }
+      } catch (error) {
+        console.error('Произошла ошибка:', error.message);
```
Comment (jszychowskilaba) [inline src/views/chat/all-rooms.html]: I think it would be better if errors are thrown in english

**Review by jszychowskilaba — CHANGES_REQUESTED** [review]

File: src/views/chat/all-rooms.html (hunk 1 above)
Comment (suprugafad) [reply src/views/chat/all-rooms.html]: @ArtemAng please translate all in eng

**Review by suprugafad — APPROVED** [review]

**Comment by jszychowskilaba** [comment]
@ArtemAng there is a pending review in order to make the merge

**Comment by suprugafad** [comment]
> @ArtemAng there is a pending review in order to make the merge

you requested changes so merging is blocked

**Review by jszychowskilaba — APPROVED** [review]

**Pull request merged**